scripts/upgrade

```diff
@@ -65,7 +65,7 @@ ynh_abort_if_errors
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 
-if [ "$upgrade_type" == "UPGRADE_APP" ]
+if [ "$upgrade_type" == "UPGRADE_APP" || "$upgrade_type" == "UPGRADE_FORCED" ] # we should do that during a force upgrade too (as it could lead to a version upgrade for instance)
```

Comment:
This is broken

```
Warning: ./upgrade: line 68: [: missing `]'
Warning: ./upgrade: line 68: UPGRADE_APP: command not found
```

```suggestion
if [ "$upgrade_type" == "UPGRADE_APP" ] || [ "$upgrade_type" == "UPGRADE_FORCED" ] # we should do that during a force upgrade too (as it could lead to a version upgrade for instance)
```

Reply:
Thanks, I thought it would work the other way around, but no ^^